README.md

```diff
@@ -36,7 +36,16 @@ julia> AL = tril!(collect(A))
  `AL` requires the same amount of storage as does `A` even though there are only 21 potential non-zeros in `AL`.
  The RFP version of the lower triangular matrix
  ```julia
- julia> ArfpL = Int.(TriangularRFP(float.(A), :L))
+ julia> ArfpL = TriangularRFP(float.(A), :L)
```

Comment:
Maybe it's simpler just to construct `A` with float elements. The `.0`s are a little noisy but then we can avoid the `Int.` calls.

Reply:
In any case should `Int.` be convering it into a regular matrix rather than modifying each entry to be an integer?

Reply:
Are you asking the the result should be a `TriangularRFP{Int}`?

Reply:
Yeah that's a better way of putting it

Reply:
I see